### PR TITLE
Add Changelog for 1.7.3

### PR DESCRIPTION
**Pull request by zeripath**
Suggested changelog for version 1.7.3

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,25 @@ This changelog goes through all the changes that have been made in each release
...
+* BUGFIXES
+  * Fix server 500 when trying to migrate to an already existing repository (#6188) (#6197)
+  * Load Issue attributes for API /repos/{owner}/{repo}/issues/{index} (#6122) (#6185)
+  * Fix bug whereby user could change private repository to public when force private enabled. (#6156) (#6165)
+  * Fix bug when update owner team then visit team's repo return 404 (#6119) (#6166)
+  * Fix heatmap and repository menu display in Internet Explorer 9+ (#6117) (#6137)
+  * Fix prohibit login check on authorization (#6106) (#6115)
+  * Move to ldap.v3 to fix #5928 (#6105) (#6107)
+  * Fix deadlock in webhook PullRequest (#6102) (#6104)
+  * modules/context/auth.go: fix redirect loop (#5965) (#6101)
```
Comment (techknowlogick) [inline CHANGELOG.md]: Can you rearrange this so that Fix... is at the start?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,11 +4,25 @@ This changelog goes through all the changes that have been made in each release
...
+  * Issue 5924 fix compare button (#5929) (#6098)
```
Comment (techknowlogick) [inline CHANGELOG.md]: Can you remove the Issue 5924 part at start?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,11 +4,25 @@ This changelog goes through all the changes that have been made in each release
...
 
+## [1.7.3](https://github.com/go-gitea/gitea/releases/tag/v1.7.3) - 2019-02-27
+* BUGFIXES
+  * Fix server 500 when trying to migrate to an already existing repository (#6188) (#6197)
+  * Load Issue attributes for API /repos/{owner}/{repo}/issues/{index} (#6122) (#6185)
+  * Fix bug whereby user could change private repository to public when force private enabled. (#6156) (#6165)
+  * Fix bug when update owner team then visit team's repo return 404 (#6119) (#6166)
+  * Fix heatmap and repository menu display in Internet Explorer 9+ (#6117) (#6137)
+  * Fix prohibit login check on authorization (#6106) (#6115)
+  * Move to ldap.v3 to fix #5928 (#6105) (#6107)
```
Comment (jolheiser) [inline CHANGELOG.md]: Should the 5928 issue be referenced here in changelog, or should it be more specific?  
`Move to ldap.v3 to fix regression bug`
or
`Fix LDAP regression bug by updating to ldap.v3`

File: CHANGELOG.md (hunk 1 above)
Comment (zeripath) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (zeripath) [reply CHANGELOG.md]: Done, now:  * Fix compare button regression (#5929) (#6098)

**Review by techknowlogick — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (zeripath) [reply CHANGELOG.md]: Now: * Fix LDAP protocol error regression by moving to ldap.v3 (#6105) (#6107)

**Review by jolheiser — APPROVED** [review]

**Pull request merged**